internal/uvm/create_lcow.go

```diff
@@ -305,6 +312,44 @@ func makeLCOWVMGSDoc(ctx context.Context, opts *OptionsLCOW, uvm *UtilityVM) (_
 		return nil, err
 	}
 
+	vmgsFile, err := os.CreateTemp("", "kernelinitrd.*.vmgs")
```

Comment:
I'm not sure it will be safe to rely on temp having sufficient space to hold everything. e.g. some Azure VMs could have small amounts of space there.

Reply:
Could we put it under the bundle dir?

Reply:
done.